### DISCUSSION TITLE
GridView.ItemClickCommand is not implemented in WinUI

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#19097] [Q&A] [answered]
```csharp
<GridView Grid.Row="3"
            ItemsSource="{x:Bind VM.Maquines, Mode=OneWay}"
            ItemTemplate="{StaticResource MachineCardNormalTemplate}"
            IsItemClickEnabled="True"
            SelectedItem="{x:Bind VM.MaquinaSeleccionada, Mode=TwoWay}"
            ItemClickCommand="{x:Bind VM.NavegarPaginaMaquinaCommand}">
</GridView>
```

`ItemClickCommand` returns this error when building WinAppSDK:

>XamlCompiler error WMC0011: Unknown member 'ItemClickCommand' on element 'GridView'

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question! There's no `ItemClickCommand` on [`GridView`](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.gridview?view=windows-app-sdk-1.6).

Yet, since `GridView` inherits from `ListViewBase`, you can use our [`CommandExtensions`](https://platform.uno/docs/articles/external/uno.toolkit.ui/doc/helpers/command-extensions.html).

**Comment by mcNets** [comment]
Just a question @jeromelaban , when `utu:CommandExtensions.Command="{x:Bind VM.NavegarPaginaMaquinaCommand}"` is fired, `SelectedItem` has not been set yet. Is there any workaround about it?
